java-manta-it/src/test/java/com/joyent/manta/http/TCPSocketConnectionTimeoutIT.java

```diff
@@ -25,19 +26,50 @@
 import java.time.Instant;
 
 /**
- * Integration tests for verifying the connection timeout behaviour for an invalid connection.
+ * Integration tests for verifying the connection timeout behaviour for an invalid connection,
+ * implies having an invalid value for MANTA_URL.
  * This Integration test is written for the sole purpose of verifying Issue #441.
  *
  * @author <a href="https://github.com/dekobon">Ashwin A Nair</a>
```

Comment:
Not that bothered about this inaccuracy, are you?

Reply:
I am not either, just felt like having additional information present there, so as to make it more obvious as to purpose of this `integration-test`. 

Generally, there have been instances(**rare**) where the SDK has had portions of code going undocumented, just wanted to improve on that, pertinent to when a fresh engineer analyzes the `SDK` code in general.

Reply:
No! I meant the link being to Elijah's github, but the name in the link being yours. The java doc are good!